neptune/internal/mockpoolutils.go

```diff
@@ -204,6 +204,10 @@ func MakeHierarchyVertex(vertexLabel, code, codeLabel string, numberOfChildren i
 	return vertex
 }
 
+func SetOrder(vertex *graphson.Vertex, order int) {
```

Comment:
Is this for internal use only? I am assuming its not a value provided by a user?

I am just wondering if there is a need to check its a sensible value? i.e. not -50000. I don't know if thats possible to determine of if it makes sense but I'll ask the question.

Reply:
This is only used for testing, in reality, the order is defined by the data team in the code list scripts, but I can add the validation here. I think that negative values would also be ok.